src/js/jsx/shared/SplitButton.jsx

```diff
@@ -52,8 +52,7 @@ define(function (require, exports, module) {
             
             var buttonIcon;
             if (this.props.iconId) {
-                buttonIcon = (<SVGIcon viewBox="0 0 16 16" CSSID={this.props.iconId}
-                    iconPath={this.props.iconPath}/>);
```

Comment:
Why don't we need this prop any longer? (Mainly I just don't understand what the use of `CSSID` is versus `iconPath`.)